packages/components/src/components/tooltip/tooltip.tsx

```diff
@@ -173,7 +173,7 @@ export class Tooltip {
     };
 
     return (
-      <div class="tooltip__container" {...eventHandlers}>
+      <div aria-label="a tooltip showing important information" aria-value={this.header} class="tooltip__container" {...eventHandlers}>
```

Comment:
i would call it additional, not important, cause tooltips always give some additional context to the element they refer to